packages/react-components/src/molecules/__tests__/TabbedCard.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { createGroupResponse } from '@asap-hub/fixtures';
+import { GroupDataObject } from '@asap-hub/model';
+import { fireEvent, render } from '@testing-library/react';
+import TabbedCard from '../TabbedCard';
+
+const groups = [
+  { ...createGroupResponse(), name: 'Group 1', id: 'g1' },
+  { ...createGroupResponse(), name: 'Group 2', id: 'g2' },
+  { ...createGroupResponse(), name: 'Group 3', id: 'g3' },
+];
+
+const firstTab = {
+  tabTitle: `First Tab`,
+  items: groups,
+  createItem: (group: GroupDataObject) => (
+    <li key={`group-${group.id}`}>{group.name}</li>
+  ),
+  truncateFrom: 2,
+};
+
+const secondTab = {
+  ...firstTab,
+  tabTitle: `Second Tab`,
+  items: [
+    {
+      ...createGroupResponse(),
+      name: 'Second tab group',
+      id: '3',
+    },
+  ],
+};
+
+it('renders the tabbed card correctly and can switch to the second tab', () => {
+  const { getByText } = render(
+    <TabbedCard
+      title="test card"
```

Comment:
These minimum props should be abstracted into const props: <minimum props> eg {...props}